frontend/public/index.html

```diff
@@ -49,7 +49,11 @@
     <link
       href="https://fonts.googleapis.com/css2?family=Noto+Sans+KR:wght@400;500;700&display=swap"
       rel="preload"
-      as="font"
+      as="style"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Noto+Sans+KR:wght@400;500;700&display=swap"
+      rel="stylesheet"
     />
```

Comment:
```suggestion
<link
      href="https://fonts.googleapis.com/css2?family=Noto+Sans+KR:wght@400;500;700&display=swap"
      rel="preload"
      as="style"
      onload=“this.onload=null; this.rel=‘stylesheet’”
    />
```

혹시 링크 두 개 대신 이렇게 해도 되는지 확인해줄 수 있나요??